#!/usr/bin/env python
PACKAGE="vn_100"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("kp_pitch",double_t,0,"pitch proportional param",10,0,30)
gen.add("ki_pitch",double_t,0,"pitch integral parameter",0.9,0,30)
gen.add("kd_pitch",double_t,0,"pitch derivative parameter",0,0,1)
gen.add("setpoint",double_t,0,"pitch setpoint",0,0,90)
gen.add("bool_param",bool_t,0,"to time the values",False)
exit(gen.generate(PACKAGE,"control","pitchparams"))